modules/veteran_verification/METADATA.yml

```diff
@@ -0,0 +1,96 @@
+openapi: '3.0.0'
+info:
+  version: 0.0.1
+  title: Veteran Verification
+  description: |
+
+    ## Background
+
+    The use-case of this API is to allow third-parties to do the following after
+    receiving authorization using an Open ID Connect flow: 
+
+    #### Veteran Status - Request confirmation from the VA of an individual's Veteran status 
+
+    The Veteran Verification API passes requests through to eMIS, the Enterprise
+    Military Information Service, and formats the response into consumable data.
+
+     #### Disability Rating - Access the disability rating of a Veteran
+
+    The Disability Rating API passes requests through to EVSS, the Electronic Veterans Self Service, and formats the response into consumable data.
+
+    #### Service History - Access the service history of a Veteran
+
+    The Service History API passes requests through to eMIS, the Enterprise
+    Military Information Service, and formats the response into consumable data.
+  
+    ## Design
+
+    ### Authorization
+
+    API requests are authorized using a Bearer token issued through an OpenID
+    Connect service to allow third-party applications. The token should be
+    submitted as an `Authorization` header in the form `Bearer <token>`.
+
+    ### Veteran Status Request
```

Comment:
Adding a section for each request schema doesn't seem very useful or sustainable. Could we make this generic and only say it once? Most of this information is also repeated in the schemas displayed below.